### DISCUSSION TITLE
Welcome to SketchIDE Discussions!

### DISCUSSION THREAD

**Discussion by rajendrakrdangi** [sketchide/SketchIDE#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SSaammiirrxuiyvfygv** [comment]
Hyyy

**Reply by rajendrakrdangi** [reply]
Yes

**Comment by aiotv1** [comment]
apk?

**Reply by rajendrakrdangi** [reply]
under development wait for initial release

**Reply by AyushVivek** [reply]
Sir but it seems that contribution in sketchide is very slow , sir.

**Comment by joniped** [comment]
I am routing for you guys still waiting for the project
